### PR TITLE
LLDB also doesn't send thread-group-exited when -target-detach is issued

### PR DESCRIPTION
**Pull request by pieandcakes**
This seems to be the same as clrdbg. Adding the same fix.
@edumunoz @rajkumar42 @chuckries

### CODE REVIEW

**Comment by edumunoz** [comment]
:ship: it.

File: src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs [hunk 1]
```diff
@@ -1381,10 +1381,10 @@ public void Terminate()
 
         public void Detach()
         {
-            // Special casing sending the fake stopped event for clrdbg. 
+            // Special casing sending the fake stopped event for clrdbg and lldb. 
             // GDB prints out thread group exit events on mi command "-target-detach" which is handed by method HandleThreadGroupExited
             // GDB or the debuggee can terminate and those are handled by Terminate and TerminateProcess methods.
-            if (MICommandFactory.Mode == MIMode.Clrdbg)
+            if (MICommandFactory.Mode == MIMode.Clrdbg && MICommandFactory.Mode == MIMode.Lldb)
```
Comment (andrewcrawley) [inline src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs]: Should be ||

**Comment by andrewcrawley** [comment]
LGTM.

**Comment by rajkumar42** [comment]
:+1:

**Pull request merged**